readthedocs/api/v3/serializers.py

```diff
@@ -472,6 +474,11 @@ class Meta:
             'programming_language',
             'repository',
             'homepage',
+            'tags',
+
+            # Advanced Settings -> General Settings
+            'default_version',
+            'default_branch',
```

Comment:
Is this related to this PR? It seems that it's adding support to change other fields besides the `tags`

Reply:
It was done because you said ` I'm fine adding support to change most (or all) the fields at this point.`  And adding other fields while creating project didn't make sense.

Reply:
Yes. I think that's fine, but we can take that work in a separate PR 👍